docusaurus/docs-internals/editor-controls/data-controls/internalLink.md

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 1
+toc_max_heading_level: 4
+---
+# Internal Link
+
+Brizy's internal link control lets you easily redirect users to any page within your website. This feature enhances navigation, improving user experience by seamlessly guiding visitors to related content or important sections.
+
+Example of the internal link:
+
+![Internal Link](/img/data-controls/internalLink.png)
+
+Example of the internal link search:
+
+![Internal Link Search](/img/data-controls/internalLinkSearch.png)
+
+Example of the internal link result:
+
+![Internal Link Result](/img/data-controls/internalLinkResult.png)
+
+### Parameters
```

Comment:
`internalLink` has also the `placeholder` props
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/7a5f9a73-3b74-46a1-97fd-19a5f004330d)

check if component can render it or how is used and if is needed then add it to parameters table